features/core.feature

```diff
@@ -13,6 +13,7 @@ Feature: Functionality of jrnl outside of actually handling journals
         Then the output should match "^jrnl version v\d+\.\d+(\.\d+)?(-(alpha|beta)\d*)?"
 
     Scenario: Running the diagnostic command
+        Given we use the config "simple.yaml"
```

Comment:
Why do we need to specify a config here? Doesn't the diagnostic command run and exit before a config is ever loaded?

Reply:
It sure does! I think I left this in to deal with a problem in the test code that I resolved later. Removing now.